microsoft-edge/devtools-guide-chromium/crash-analyzer/index.md

```diff
@@ -0,0 +1,255 @@
+
+![The Crash analyzer tool](./index-images/crash-analyzer.png)
+
+When a web app crashes or hangs unexpectedly, this can result in data loss and a poor experience for your users.  Collecting information about the crashes that your users experience is helpful to diagnose and fix the root cause of the crashes. However, code that's running in production is often minified, which causes the JavaScript stack traces that are logged on crashes to also be minified.  The **Crash analyzer** tool helps you map minified stack traces collected in production back to the original source code, so that you can quickly identify the root cause of the crashes.
+
+Here is an example of a minified stack trace that you might collect from your web app in production:
+
+```
+Uncaught Error: test error
+    at https://microsoftedge.github.io/Demos/devtools-crash-analyzer/app/prod.bundle.js:1:1344
+    at t.render (https://microsoftedge.github.io/Demos/devtools-crash-analyzer/app/prod.bundle.js:1:1368)
+    at https://microsoftedge.github.io/Demos/devtools-crash-analyzer/app/prod.bundle.js:1:2278
+    at Array.map (<anonymous>)
+    at t.render (https://microsoftedge.github.io/Demos/devtools-crash-analyzer/app/prod.bundle.js:1:2252)
+    at new t (https://microsoftedge.github.io/Demos/devtools-crash-analyzer/app/prod.bundle.js:1:2063)
+    at https://microsoftedge.github.io/Demos/devtools-crash-analyzer/app/prod.bundle.js:1:2857
+    at https://microsoftedge.github.io/Demos/devtools-crash-analyzer/app/prod.bundle.js:1:2896
+```
+
+Variable and function names are often shortened, like the variable `t` above. Script URLs often point to a built single-line bundle file, like the `prod.bundle.js` file above.
+
+By comparison, here is an example of an unminified stack trace, with original variable and function names, and original source file names and line numbers:
+
+```
+Uncaught Error: test error
+    at shorten (util.ts:9:9)
+    at processUIString (util.ts:2:10)
+    at todoMarkupBuilder (todo.ts:35:41)
+    at Todo.render (todo.ts:24:12)
+    at app.ts:36:39
+    at Array.map (<anonymous>)
+    at App.render (app.ts:36:22)
+    at new App (app.ts:29:10)
+    at index.ts:9:13
+    at index.ts:9:33
+```
+
+There are two steps to analyze minified production stack traces in the **Crash analyzer** tool:
+
+1. First, you collect specially formatted stack traces from your web app in production.
+
+   There are many ways to collect stack traces from the browsers your users are using. In [Collect stack traces in production](#collect-stack-traces-in-production), below, we provide examples and recommendations. We also explain how to format the stack traces so that they can be analyzed in the **Crash analyzer** tool.
```

Comment:
```suggestion
   There are many ways to collect stack traces from the browsers your users are using. The section [Collect stack traces in production](#collect-stack-traces-in-production), below, provides examples and recommendations. It also explains how to format the stack traces so that they can be analyzed in the **Crash analyzer** tool.
```
moving focus away from us onto docs. 
1. put focus on the tech.
2. put focus on the docs.
3. put focus on us.

corp style guide advises against "we".  
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we - "In general, don't use [we]. Focus on the customer, and avoid making Microsoft the subject.  It's OK to use phrasing like _we recommend_ if it helps you avoid awkward phrasing like _it's recommended_, but write around it if you can."

@captainbrosset